documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -91,7 +94,7 @@ kubectl logs -f _CLUSTER-OPERATOR-POD-NAME_ | grep -E "Kafka version upgrade fro
 +
 [source,shell,subs=+quotes]
 ----
-kubectl get pod my-cluster-kafka-0 -o jsonpath='{.spec.containers[0].image}'
+kubectl get pods my-cluster-kafka-0 -o jsonpath='{.spec.containers[0].image}'
 ----
 +
 If the current and new versions of Kafka have different inter-broker protocol versions, check the Cluster Operator logs for an `INFO` level message:
```

Comment:
The message on lines 104 and 110 do not exist anymore and should be removed.

Reply:
Not done. I'd rather change this in a subsequent PR for multi-version upgrades.